core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -326,6 +326,38 @@ static function (array $value): array {
                     ->prototype('scalar')->end()
                     ->defaultValue(['jpg', 'jpeg', 'gif', 'png', 'tif', 'tiff', 'bmp', 'svg', 'svgz', 'webp'])
                 ->end()
+                ->arrayNode('preview')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('target_dir')
```

Comment:
Something we might want to discuss related to #3774: 

With VFSs in place, we could/should probably deprecate all configurable paths and suggest to use mounting instead. This would include the image assets path, the upload dir, this new config and possibly more.

The current default (whatever it is `assets/images` or `var/images` …) would then become just an identifier. If we wanted to switch locations later on, we could use mounting ourselves. (The implementation would get a `VirtualFilesystem` prefixed with the identifier and would not care about the location at all.)